chapter/templates/chapter.tex

```diff
@@ -2,5 +2,18 @@ \chapter{<%= chapterName %>}
 
 Write <%= chapterName %> here.
 Example of a glossary entry for the word \gls{computer}.
-Example of an acronym entry with a latin saying \gls{qed}. Notice how it is
-shorten the second time, \gls{qed}.
+
+Example of an acronym entry with a latin saying: \gls{qed}. Notice how it is
+shorten the second time: \gls{qed}.
+
+Figure \ref{figure_example} gives an example of an svg graphic converted automatically to .pdf and
+imported in latex.
+
+\begin{figure}[h]
```

Comment:
Yes, README. In fact, if you want to have an implementation somewhere for testing purposes, see test files. 
Le 7 juin 2016 13:36, Rémi Bèges <notifications@github.com> a écrit :In chapter/templates/chapter.tex:
> @@ -2,5 +2,18 @@ \chapter{<%= chapterName %>}
>  
>  Write <%= chapterName %> here.
>  Example of a glossary entry for the word \gls{computer}.
> -Example of an acronym entry with a latin saying \gls{qed}. Notice how it is
> -shorten the second time, \gls{qed}.
> +
> +Example of an acronym entry with a latin saying: \gls{qed}. Notice how it is
> +shorten the second time: \gls{qed}.
> +
> +Figure \ref{figure_example} gives an example of an svg graphic converted automatically to .pdf and
> +imported in latex.
> +
> +\begin{figure}[h]

Good point, would only putting this in the first chapter be good for you ? Or do you still prefer the readme approach?

—You are receiving this because you commented.Reply to this email directly, view it on GitHub, or mute the thread.